pandas/core/generic.py

```diff
@@ -3636,6 +3637,34 @@ def astype(self, dtype, copy=True, errors='raise', **kwargs):
         Returns
         -------
         casted : type of caller
+
+        Examples
```

Comment:
Ok, done, added example with ``copy=False``, where result propagates upwards.

I could only get it to work with categoricals and not numpy dtypes, so the example is a bit contrieved.